src/Features/Core/Portable/CodeCleanup/CodeCleanupOptions.cs

```diff
@@ -17,6 +17,15 @@ internal static class CodeCleanupOptions
             nameof(CodeCleanupOptions), nameof(NeverShowCodeCleanupInfoBarAgain), defaultValue: false,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Never Show Code Cleanup Info Bar Again"));
 
+        /// <summary>
+        /// Set default value to true for 15.8 Preview to promote this new code cleanup feature
+        /// Will set it back to false for RTM so users are not silently opting into this feature
+        /// https://github.com/dotnet/roslyn/issues/28068
+        /// </summary>
+        public static readonly PerLanguageOption<bool> PerformAdditionalCodeCleanupDuringFormatting = new PerLanguageOption<bool>(
```

Comment:
shouldn't this be false by default? @jinujoseph is the last decision making this off by default?